packages/integration-test-app/runner/lib/IntegrationTestRunner.ts

```diff
@@ -82,7 +82,7 @@ export default class IntegrationTestRunner {
                 }:${frame.column})`,
             )
             .join();
-        fail(err);
+        throw err;
         break;
```

Comment:
Change made to replace `fail()` call which appears to no longer be available: https://github.com/jestjs/jest/issues/11698